rubygem-newt/rubygem-newt.spec

```diff
@@ -0,0 +1,75 @@
+%if 0%{?rhel} == 6
+%{!?gem_extdir_mri:%global gem_extdir_mri %(ruby -rrbconfig -e "puts Config::CONFIG['sitearchdir']")}
+%global gem_extdir_mri_lib %{gem_extdir_mri}
+%else
+%{!?gem_extdir_mri:%global gem_extdir_mri %{gem_extdir}}
+%global gem_extdir_mri_lib %{gem_extdir_mri}/lib
+%endif
+%global gem_name newt
+
+Summary: Ruby bindings for newt
+Name: rubygem-%{gem_name}
+
+Version: 0.9.4
+Release: 1%{?dist}
+Group: Development/Languages
+License: MIT
+URL: https://github.com/theforeman/ruby-newt
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Requires: ruby
+Requires: rubygems
+
+Requires: newt
+
+BuildRequires: ruby
+BuildRequires: rubygems
+BuildRequires: rubygems-devel
+BuildRequires: ruby-devel
+
+BuildRequires: newt-devel
+
+Provides: rubygem(%{gem_name}) = %{version}
+
+%description
+Ruby bindings for the newt TUI library allows creation of rich
+user text-based experience in Ruby language.
+
+%package doc
+BuildArch:  noarch
+Requires:   rubygem-%{gem_name} = %{version}-%{release}
+Summary:    Documentation for rubygem-%{gem_name}
+
+%description doc
+This package contains documentation for rubygem-%{gem_name}.
+
+%prep
+%setup -q -c -T
+mkdir -p .%{gem_dir}
+gem install --local --install-dir .%{gem_dir} \
+            --force %{SOURCE0}
+find .%{gem_dir}
+
+%build
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+mkdir -p %{buildroot}%{gem_extdir_mri_lib}/ruby_newt
+mv %{buildroot}%{gem_instdir}/lib/ruby_newt/ruby_newt.so %{buildroot}%{gem_extdir_mri_lib}/ruby_newt/ruby_newt.so
+rm -rf %{buildroot}%{gem_dir}/gems/%{gem_name}-%{version}/{ext,tmp,.require_paths}
```

Comment:
`%{gem_dir}/gems/%{gem_name}-%{version}` == `%{gem_instdir}` or just use %exclude in the files list